### PR TITLE
Feature TG Projects

### PR DESCRIPTION
**Pull request by ArianaMaldonado**
This PR contains the following functions for the **Projects** section:

- Get all projects (to obtain the list of projects).
- Get by id (to obtain a project by id).
- Post a new project (to add a project to the list).

### CODE REVIEW

**Comment by ArianaMaldonado** [comment]
This PR contains the following functions for the **Projects** section:

- Get all projects (to obtain the list of projects).
- Get by id (to obtain a project by id)
- Post a new project (to add a project to the list)

File: src/data/projects.json [hunk 1]
```diff
@@ -0,0 +1 @@
... non sodales sed tincidunt eu felis fusce posuere felis","startDate":"12/9/2021","endDate":"9/6/2022","clientName":"Isabelita Trewhella","active":false,"employee":[{"id":3,"role":"DEV","rate":10},{"id":26,"role":"PM","rate":10},{"id":38,"role":"QA","rate":3},{"id":5,"role":"PM","rate":6}],"admin":"Dacy Febry"},{"id":"9","name":"Helena Quare","description":"ut massa volutpat convallis morbi odio odio elementum eu interdum eu tincidunt in leo","startDate":"12/28/2021","endDate":"8/24/2022","clientName":"Marchall Duffan","active":false,"employee":[{"id":38,"role":"QA","rate":5},{"id":42,"role":"PM","rate":6},{"id":25,"role":"PM","rate":3}],"admin":"Hans Crewe"},{"id":"10","name":"Geri Sier","description":"sed lacus morbi sem mauris laoreet ut rhoncus aliquet pulvinar","startDate":"8/24/2021","endDate":"12/12/2023","clientName":"Darla Ondrusek","active":true,"employee":[{"id":36,"role":"PM","rate":10},{"id":4,"role":"DEV","rate":9},{"id":26,"role":"QA","rate":10}],"admin":"Payton Fulstow"},{"id":11,"description":"Here should be the description of the project.","startDate":"4/12/2022","endDate":"8/25/2022","clientName":"Radium Rocket","employee":[{"id":"32","role":"DEV","rate":"4.5"}]}]
```
Comment (cocolobos) [inline src/data/projects.json:1]: We should give the correct format to the JSON file

File: src/index.js [hunk 2]
```diff
@@ -17,6 +18,12 @@ app.get('/admins', (req, res) => {
   });
 });
 
+// Projects
+
+const projectsRouter = require('./resources/projects');
```
Comment (cocolobos) [inline src/index.js:23]: We should set all variables at the begining of the file and the separate the functions by entity

File: src/index.js [hunk 3]
```diff
@@ -17,6 +18,12 @@ app.get('/admins', (req, res) => {
...
+
+app.use('/projects', projectsRouter);
```
Comment (cocolobos) [inline src/index.js:25]: This line should be below the port listener

File: src/resources/projects.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
+const express = require('express');
+const fs = require('fs');
+
```
Comment (cocolobos) [inline src/resources/projects.js:3]: remove empty space

File: src/resources/projects.js [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+const projects = require('../data/projects.json');
+
```
Comment (cocolobos) [inline src/resources/projects.js:5]: remove empty space

Comment (ArianaMaldonado) [reply src/resources/projects.js:5]: Eslint me lo vuelve a poner cada vez que lo borro.

**Review by cocolobos — APPROVED** [review]

**Pull request merged**